OMCompiler/Compiler/SimCode/SimCodeMain.mo

```diff
@@ -961,12 +973,22 @@ algorithm
 
         // Add external libraries and includes
         cmakelistsStr := System.stringReplace(cmakelistsStr, "@FMI_INTERFACE_HEADER_FILES_DIRECTORY@", "\"" + Settings.getInstallationDirectoryPath() + "/include/omc/c/fmi" + "\"");
-        (needCvode, cvodeDirectory) := SimCodeUtil.getCmakeSundialsLinkCode(simCode.fmiSimulationFlags);
-        cmakelistsStr := System.stringReplace(cmakelistsStr, "@NEED_CVODE@", needCvode);
-        cmakelistsStr := System.stringReplace(cmakelistsStr, "@CVODE_DIRECTORY@", cvodeDirectory);
+        (solverNameCS, solverLibDirectory) := SimCodeUtil.getCmakeSundialsLinkCode(simCode.fmiSimulationFlags);
+        cmakelistsStr := System.stringReplace(cmakelistsStr, "@SOLVER_NAME@", "\"" + solverNameCS + "\"");
+        cmakelistsStr := System.stringReplace(cmakelistsStr, "@SOLVER_DIRECTORY@", solverLibDirectory);
         cmakelistsStr := System.stringReplace(cmakelistsStr, "@FMU_ADDITIONAL_LIBS@", SimCodeUtil.getCmakeLinkLibrariesCode(simCode.makefileParams.libs));
         cmakelistsStr := System.stringReplace(cmakelistsStr, "@FMU_ADDITIONAL_INCLUDES@", SimCodeUtil.make2CMakeInclude(simCode.makefileParams.includes));
 
+//Do we need to write codes as above for IDA?
```

Comment:
I added `getCmakeSundialsLinkCode` here and with that the CMakeLists.txt is generated as expected.